plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -954,4 +958,13 @@ public static long getModificationTime(String path, TrinoFileSystem fileSystem)
             throw new TrinoException(ICEBERG_FILESYSTEM_ERROR, "Failed to get file modification time: " + path, e);
         }
     }
+
+    public static Map<String, String> loadManifestCachingProperties(Map<String, String> properties, IcebergConfig icebergConfig)
+    {
+        properties.put(IO_MANIFEST_CACHE_ENABLED, "true");
+        properties.put(IO_MANIFEST_CACHE_MAX_TOTAL_BYTES, String.valueOf(icebergConfig.getMaxManifestCacheSize().toBytes()));
```

Comment:
How big is the default?

Reply:
By default it is 100MB, integrated in the Iceberg library:  
![image](https://github.com/trinodb/trino/assets/26461591/3404248a-24c5-4c0e-8660-d2b76d4c3a01)